src/libraries/Microsoft.PowerFx.Interpreter/MaxCallDepthException.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Interpreter
+{
+    [Serializable]
+    internal class MaxCallDepthException : Exception
```

Comment:
>MaxCallDepthException [](http://example.com/codeflow?start=19&length=21)

nit: I'd add a comment that this is an interpreter internal excpetion and should be converted to a ErrorValue before returning to the host. 

Even consider adding the "convert to error value" as a helper method on this class to make that very clear.

Reply:
Done.